lib/actions/field-trip.js

```diff
@@ -122,22 +206,23 @@ export function fetchFieldTripDetails (requestId, intl) {
 /**
  * Add note for field trip request.
  */
-export function addFieldTripNote (request, note, intl) {
+export function addFieldTripNote(request, note, intl) {
   return function (dispatch, getState) {
-    const {callTaker, otp} = getState()
-    const {datastoreUrl} = otp.config
+    const { callTaker, otp } = getState()
+    const { datastoreUrl } = otp.config
     if (sessionIsInvalid(callTaker.session)) return
-    const {sessionId, username} = callTaker.session
+    const { sessionId, username } = callTaker.session
     const noteData = serialize({
-      note: {...note, userName: username},
+      note: { ...note, userName: username },
       requestId: request.id,
       sessionId
     })
-    return fetch(`${datastoreUrl}/fieldtrip/addNote`,
-      {body: noteData, method: 'POST'}
-    )
+    return fetch(`${datastoreUrl}/fieldtrip/addNote`, {
```

Comment:
Is this fetch not able to use the new request method?

Reply:
Handled in https://github.com/opentripplanner/otp-react-redux/pull/627/commits/42a8fb2c0864378234354ceb9827ddf380ac03bc.